04/r/particulate-matter.Rmd

```diff
@@ -0,0 +1,385 @@
+ggplot(data, aes(x = log_sat, y = log_pm25)) +
+    geom_point() +
+    geom_smooth(method = "lm")
+```
+
+If you *squint*, the linear regression line seems like an OK(ish) place to start; the data definitely has an overall linear correlation. The focus of the rest of this notebook will be on the extent to which deviations from this linear relationship are going to mess us up.
+
+If we look closer, however, the errors don't seem to be totally random (e.g. they're not I.I.D. normal) - the data seems to have some structure to it, with some areas tending to have positive residuals and other areas negative. 
+
+We can visualize this easier by plotting the difference between `log_pm25` and `log_sat` on the y-axis instead of just `log_pm25` by itself:
+```{r plot-ground-vs-satellite-residuals}
+ggplot(data, aes(x = log_sat, y = log_pm25 - log_sat)) +
+    geom_point() +
+    geom_smooth(method = "lm") +
+    geom_hline(yintercept = 0, color = "red", linetype = "dashed")
+```
+
+If the errors were IID we would expect them to be normally distributed around 0 (the red dashed line). But there is clearly a non-random pattern here. Let's do some more exploration to see if we can (qualitatively) understand where this is coming from.
+
+One possibility is that different types of locations have different relationships between satellite and ground station data. We could do a quick check by breaking out the above plot by WHO super-regions:
+
+```{r plot-ground-vs-satellite-facet-region}
+ggplot(data, aes(x = log_sat, y = log_pm25, color = super_region_name)) +
+    geom_point(alpha = 0.3) +
+    geom_smooth(method = "lm", se = FALSE)
+```
+
+Notice that the different geographic areas only partially overlap, and regression lines fit to each show very different relationships. Gabry *et al* also tried grouping stations using hierarchical clustering (labeled `cluster_region` in the dataset):
+
+```{r plot-ground-vs-satellite-facet-cluster}
+ggplot(data, aes(x = log_sat, y = log_pm25, color = cluster_region)) +
+    geom_point(alpha = 0.3) +
+    geom_smooth(method = "lm", se = FALSE)
+```
+
+Just as in the super-region plot, we see very different dependences here. When we start modeling, we'll need to be on the lookout for biases in predictions on different types of locations.
+
+## 3. Baseline Model Prior
+To have something to benchmark against, let's start by fitting a Bayesian linear regression,
+
+$$
+y_i \sim N(\beta_0 + \beta_1x_i, \sigma^2)
+$$
+where $y_i$ is the ground station $PM_{2.5}$ measurement, $x_i$ is this satellite measurement, and $\beta_0$, $\beta_1$, and $\sigma$ are the parameters we need to infer. We'll use a normal distribution for priors on $\beta_0$ and $\beta_1$, and a half-normal for $\sigma$ (since it has to be positive).
+
+Before fitting anything, we should do a *keep yourself honest* check to see how well we've specified our model. Even without touching stan, it's easy to draw fake data from the prior predictive distribution:
```

Comment:
But we do use stan here (which i think it wasn't in the original version). we should change that language. It might also be useful to show how to check our prior with a simpler method.